src/wallet/rpcwallet.cpp

```diff
@@ -3684,23 +3695,24 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     if (fHelp || params.size() < 2 || params.size() > 4)
         throw runtime_error(
             "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf ) ( fee )\n"
-            "\nSend multiple times. Amounts are decimal numbers with at most 8 digits of precision."
-            "\nChange generated from one or more transparent addresses flows to a new transparent"
-            "\naddress, while change generated from a shielded address returns to itself."
+            "\nSend multiple times. Amounts are decimal numbers with at most 8 digits of precision. "
+            "\nChange will be sent to the unified change address of the HD account associated with "
+            "\n`fromaddress` or, if `fromaddress` specifies a legacy address not derived from the "
+            "\nwallet's HD seed, change will be sent to the unified change address corresponding to "
+            "\nthe legacy account id 0x7FFFFFFF.\n"
             "\nWhen sending coinbase UTXOs to a shielded address, change is not allowed."
             "\nThe entire value of the UTXO(s) must be consumed."
-            + strprintf("\nBefore Sapling activates, the maximum number of zaddr outputs is %d due to transaction size limits.\n", Z_SENDMANY_MAX_ZADDR_OUTPUTS_BEFORE_SAPLING)
             + HelpRequiringPassphrase() + "\n"
             "\nArguments:\n"
-            "1. \"fromaddress\"         (string, required) The transparent or shielded address to send the funds from.\n"
+            "1. \"fromaddress\"         (string, required) The transparent, Sprout, Sapling, or Unified address to send the funds from.\n"
             "                           The following special strings are also accepted:\n"
             "                               - \"ANY_TADDR\": Select non-coinbase UTXOs from any transparent addresses belonging to the wallet.\n"
             "                                              Use z_shieldcoinbase to shield coinbase UTXOs from multiple transparent addresses.\n"
             "2. \"amounts\"             (array, required) An array of json objects representing the amounts to send.\n"
             "    [{\n"
-            "      \"address\":address  (string, required) The address is a taddr or zaddr\n"
-            "      \"amount\":amount    (numeric, required) The numeric amount in " + CURRENCY_UNIT + " is the value\n"
-            "      \"memo\":memo        (string, optional) If the address is a zaddr, raw data represented in hexadecimal string format\n"
+            "      \"address\":address  (string, required) The transparent, Sapling, or Unified address of the recipient\n"
```

Comment:
This can be a Sprout address.

Reply:
Aren't we entirely disabling *->sprout, except for interstitial trees? Is there any reason to continue supporting sprout->sprout at the RPC interface level?

Reply:
No, we aren't disabling `* -> Sprout` in NU5; `Sprout -> Sprout` is still allowed at the consensus level.

Reply:
After discussion in Slack, we have decided to remove wallet support for sending to Sprout with the exception that change may be returned to a Sprout address when sending funds from Sprout.